styles/main.css

```diff
@@ -244,8 +244,8 @@ footer {
   box-shadow: none;
   width: 20%;
   position: absolute;
-  right: 0px;
-  top: 0px;
+  right: -7px;
+  top: -7px;
```

Comment:
You can set your position to negative values to move them up and farther to the right!